catalyst/dl/callbacks/mixup.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+from catalyst.dl.core.state import RunnerState
+from catalyst.dl.callbacks import CriterionCallback
+import numpy as np
+import torch
+
+
+class MixupCallback(CriterionCallback):
+    def __init__(self, fields: List[str] = ('features',), alpha=1.0, train_only=True, **kwargs):
+        assert alpha >= 0, 'alpha must be>=0'
+
+        super(MixupCallback, self).__init__(**kwargs)
```

Comment:
for py3 you can use just `super().__init__` to follow lib code standarts

Reply:
That is great information, thanks. Corrected